.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -15,6 +15,7 @@ assignees: ''
 - [ ] `@azure/msal-angular@0.x.x`
 - [ ] `@azure/msal-angular@1.x.x`
 - [ ] `@azure/msal-angularjs@1.x.x`
+- [ ] `@azure/msal-node@1.x.x-beta.x`
```

Comment:
Small thing: I would say we don't need to have the prerelease names include, given someone may still be using alpha, and so that we don't have to update against once GA.

```suggestion
- [ ] `@azure/msal-node@1.x.x`
```

Reply:
@sameerag @hectormmg  What do you think?  Since you proposed using the beta designation?  

I don't personally think this will affect the qualify of the submissions or our ability to address them.  I'm open to either choice or just saying ...msal-node@beta